scripts/matching-profile/join-with-locations

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+while read -r ordinal time; do
+  loc=$(llvm-kompile-compute-loc "$2" $ordinal)
+  echo "${loc##*/} $time"
+done < "$1"
```

Comment:
These are just some basic one-off data processing scripts I wrote that I didn't want to lose track of because they might be useful in the future. I will add comments explaining what each one does, but I don't want to spend more time than is needed on one off scripts, and adding command line parsing would make them a lot more complicated.